src/resolve-script-name.js

```diff
@@ -0,0 +1,28 @@
+import {get} from 'lodash'
+
+export default resolveScriptName
+
+function resolveScriptName(scriptToFind, scriptsConfig) {
```

Comment:
I understand, but I actually think the way it is, it is better. Because I don't see a way to get resolved script name by his prefix in other way than reimplementing `prefix-matcher` 😄 I may be wrong, correct me if you see other solution.